system-addon/test/unit/lib/PlacesFeed.test.js

```diff
@@ -244,11 +244,7 @@ describe("PlacesFeed", () => {
         // Yes, onItemAdded has at least 8 arguments. See function definition for docs.
         const args = [null, null, null, TYPE_BOOKMARK,
           {spec: FAKE_BOOKMARK.url, scheme: "http"}, FAKE_BOOKMARK.bookmarkTitle,
-          FAKE_BOOKMARK.dateAdded,
-          FAKE_BOOKMARK.bookmarkGuid,
-          "",
-          SOURCES.DEFAULT
-        ];
+          FAKE_BOOKMARK.dateAdded, FAKE_BOOKMARK.bookmarkGuid, "", SOURCES.DEFAULT];
```

Comment:
Usually I'm of "all on same or all on separate" but this is for testing with a bunch of `null`s and seems to wrap okay